endpoints/publicProxy/http.go

```diff
@@ -337,7 +363,9 @@ func SetZrokCookie(w http.ResponseWriter, domain, email, accessToken, provider s
 		Domain:  domain,
 		Path:    "/",
 		Expires: time.Now().Add(checkInterval),
-		//Secure:  true, //When tls gets added have this be configured on if tls
+		// Secure:  true, //When tls gets added have this be configured on if tls
```

Comment:
dead code -- just remove?

Reply:
I'll expound on what, why, and when when I finish auditing cookie options